src/smriprep/cli/run.py

```diff
@@ -115,6 +134,17 @@ def get_parser():
         '{<suffix>:{<entity>:<filter>,...},...} '
         '(https://github.com/bids-standard/pybids/blob/master/bids/layout/config/bids.json)',
     )
+    g_bids.add_argument(
+        '--subject-anatomical-reference',
+        choices=['first-alpha', 'unbiased', 'sessionwise'],
+        default='first',
+        help='Method to produce the reference anatomical space:'
+        '\t"first-alpha" will use the first alphabetically sorted image'
+        '\t"unbiased" will construct an unbiased template from all images '
+        '(previously "--longitudinal")'
+        '\t"sessionwise" will independently process each session. If multiple sessions are '
+        'found, the behavior will be similar to "first"',
```

Comment:
```suggestion
        '\t"sessionwise" will independently process each session. If multiple sessions are '
        'found, the behavior will be similar to "first-alpha"',
```

Wouldn't the behavior match `first-alpha` if _one_ session is found, rather than multiple sessions?